pm_serde_example/pm_read.c

```diff
@@ -8,7 +8,6 @@ int main (int argc, char** argv)
 {
   OPObject* list;
   FILE* in = fopen("list_serialized", "r");
-  PMMemoryManager* ctx = PMDeserialize(in, &list);
```

Comment:
Without this line program wouldn't work.
We should free the manager in the end instead.